src/core/Typography/Text/Text.tsx

```diff
@@ -0,0 +1,96 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import { useTheme } from '../../utils/hooks/useTheme';
+import { CommonProps } from '../../utils/props';
+import '@itwin/itwinui-css/css/text.css';
+
+type TextOwnProps<T extends React.ElementType | React.ComponentType = 'div'> = {
+  /**
+   * What element should the text be rendered as?
+   *
+   * @default 'div'
```

Comment:
If default variant is `body`, then default element should be `<p>`

Reply:
`<p>` doesn't make sense to me as a default. Text can be anywhere, and it's not usually a paragraph. I could go with `<span>`.

Reply:
I was also thinking about `<span>` but as `body` is default variant and it is `<p>`.